docs/concepts/overview/components.md

```diff
@@ -59,74 +59,60 @@ The following controllers have cloud provider dependencies:
-
-#### DNS
+Node components run on every node, maintaining running pods and providing the Kubernetes runtime environment.
 
-While the other addons are not strictly required, all Kubernetes clusters should have [cluster DNS](/docs/concepts/services-networking/dns-pod-service/), as many examples rely on it.
+### kubelet
 
-Cluster DNS is a DNS server, in addition to the other DNS server(s) in your environment, which serves DNS records for Kubernetes services.
+{% glossary_definition term_id="kubelet" length="all" %}
 
-Containers started by Kubernetes automatically include this DNS server in their DNS searches.
+### kube-proxy
 
-#### Web UI (Dashboard)
+[kube-proxy](/docs/admin/kube-proxy/) enables the Kubernetes service abstraction by maintaining
+network rules on the host and performing connection forwarding.
 
-[Dashboard](/docs/tasks/access-application-cluster/web-ui-dashboard/) is a general purpose, web-based UI for Kubernetes clusters. It allows users to manage and troubleshoot applications running in the cluster, as well as the cluster itself.
+### Container Runtime
 
+Container Runtime is a software that is responsible for running containers.
 
-#### Container Resource Monitoring
+#### docker
+[Docker](http://www.docker.com) is a platform that can be used for the above purpose.
 
-[Container Resource Monitoring](/docs/tasks/debug-application-cluster/resource-usage-monitoring/) records generic time-series metrics
-about containers in a central database, and provides a UI for browsing that data.
-
-#### Cluster-level Logging
+#### rkt
 
-A [Cluster-level logging](/docs/concepts/cluster-administration/logging/) mechanism is responsible for
-saving container logs to a central log store with search/browsing interface.
+[rkt](https://coreos.com/rkt/) is supported experimentally as an alternative to docker.
 
-## Node components
+## Addons
 
-Node components run on every node, maintaining running pods and providing the Kubernetes runtime environment.
-
-### kubelet
-
-[kubelet](/docs/admin/kubelet/) is the primary node agent. It watches for pods that have been assigned to its node (either by apiserver or via local configuration file) and:
+Addons are pods and services that implement cluster features. The pods may be managed
+by Deployments, ReplicationControllers, and so on. Namespaced addon objects are created in
+the `kube-system` namespace.
 
-  * Mounts the pod's required volumes.
-  * Downloads the pod's secrets.
-  * Runs the pod's containers via docker (or, experimentally, rkt).
-  * Periodically executes any requested container liveness probes.
-  * Reports the status of the pod back to the rest of the system, by creating a *mirror pod* if necessary.
-  * Reports the status of the node back to the rest of the system.
+Addon manager creates and maintains addon resources. Selected addons are described below, for extended list of available addons please see [Addons](/docs/concepts/cluster-administration/addons/) page.
```

Comment:
I like where you're going with this update, but the concept of an "Addon manager" sounds like code running somewhere that you can interact with, yet that reference doesn't exist elsewhere in the docs. I've seen that structure on `minikube`, but as far as I can tell that's the only "addon manager" that coordinates this effort. 

Could you expand a bit on the addon manager if you're going to reference it, and describe reference how you install it and work with it? Otherwise I think it might be better to strike this line, as it sets up expectations that this mechanism exists but there's no documentation for it.

Reply:
Thank you for this catch. You're right this sentence need to be removed, Addons manager shouldn't be mentioned here.